api/src/main/java/org/pmiops/workbench/db/dao/UserRecentResourceService.java

```diff
@@ -31,4 +32,6 @@ DbUserRecentResource updateNotebookEntry(
   void deleteCohortReviewEntry(long workspaceId, long userId, long cohortReviewId);
 
   List<DbUserRecentResource> findAllResourcesByUser(long userId);
```

Comment:
It would be helpful to mark this method as `@Deprecated`, along with the DbUserRecentResource class and DAO